packages/core/services/managers/marker-manager.ts

```diff
@@ -4,18 +4,15 @@ import { Observable, Observer } from 'rxjs';
 import { AgmMarker } from './../../directives/marker';
 
 import { GoogleMapsAPIWrapper } from './../google-maps-api-wrapper';
-import { Marker } from './../google-maps-types';
-
-declare var google: any;
 
 @Injectable()
 export class MarkerManager {
-  protected _markers: Map<AgmMarker, Promise<Marker>> =
-      new Map<AgmMarker, Promise<Marker>>();
+  protected _markers: Map<AgmMarker, Promise<google.maps.Marker>> =
+      new Map<AgmMarker, Promise<google.maps.Marker>>();
 
   constructor(protected _mapsWrapper: GoogleMapsAPIWrapper, protected _zone: NgZone) {}
 
-  async convertAnimation(uiAnim: 'BOUNCE' | 'DROP' | null): Promise<any>{
+  async convertAnimation(uiAnim: keyof typeof google.maps.Animation | null): Promise<any>{
```

Comment:
we should drop this method completely, since we are now using the enum

Reply:
Can we do that as part of a separate PR? (That and whatever things should be deleted)

Reply:
this method was there only because of lack of types.